### PR TITLE
docs: add instructions to update release docs

### PR DESCRIPTION
**Pull request by sdowell**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
 1. Click `Publish release` button.
 1. Send an announcement email in the [kpt users google group].
 
+## Updating function docs
```
Comment (yuwenma) [inline RELEASING.md:17]: To follow our own codelab convention, can you use an example to guide through the steps?
e.g.

The RELEASE_BRANCH branch should already exist in repo https://github.com/GoogleContainerTools/kpt-functions-catalog. It is in the form of ${FUNCTION_NAME}/v${MAJOR_VERSION}.${MINOR_VERSION}. For example, `set-namespace/v0.3` `kubeval/v0.1`

File: RELEASING.md [hunk 2]
```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
...
+
+1. Run the script:
+```shell
+# Check out latest version of the make target from master
+git fetch upstream && git checkout upstream/master
+# Run the make target
+# e.g. RELEASE_BRANCH=set-namespace/v0.3 make update-function-docs
+RELEASE_BRANCH=<RELEASE_BRANCH> make update-function-docs
+```
+2. The script will generate a new commit in your local repository which updates
```
Comment (yuwenma) [inline RELEASING.md:30]: nit: Use `1.` here and it will be shown in the list form. See example above

File: RELEASING.md [hunk 3]
```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
...
+the docs for the provided function release. Push this commit and create a pull
+request targeted at the release branch.
```
Comment (yuwenma) [inline RELEASING.md:32]: Can you highlight these two questions:
1. running this make rule will switch branches, can you tell the source-branch (HEAD detached at upstream/master) to the local release branch (${RELEASE_BRANCH}. qq: shall user delete the local release-branch or make the local-release branch be up to date with the remote upstream RELEASE_BRANCH?
2. The example command user shall run

Comment (sdowell) [reply RELEASING.md]: Yeah it seems there is an assumption that the local release branch is up-to-date with remote. I added a step to ensure that this is the case. Alternatively the script itself could probably handle checking out the remote, although the usage of the local branch is useful for testing

File: RELEASING.md (hunk 2 above)
Comment (sdowell) [reply RELEASING.md]: Done

File: RELEASING.md (hunk 1 above)
Comment (sdowell) [reply RELEASING.md:17]: Done

**Review by yuwenma — APPROVED** [review]

**Pull request merged**